transformers/integration_tests/cat_file_test.go

```diff
@@ -203,6 +204,46 @@ var _ = Describe("Cat File transformer", func() {
 			ContractAddresses: []string{test_data.Cat110Address()},
 			ContractAbi:       constants.Cat110ABI(),
 		}
```

Comment:
Not sure what we want to do here. Do we leave out a test for `chop_lump` since its unchanged between versions?